binder/nanoevents.ipynb

```diff
@@ -31,15 +31,24 @@
    "cell_type": "code",
    "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/saransh/Code/HEP/coffea/src/coffea/nanoevents/schemas/nanoaod.py:243: RuntimeWarning: Missing cross-reference index for FatJet_genJetAK8Idx => GenJetAK8\n",
```

Comment:
I think I should suppress this warning.

Reply:
set `NanoAODSchema.warn_missing_crossrefs = False` before you make nanoevents.

Reply:
Thanks!